README.md

```diff
@@ -1,33 +1,83 @@
-# Sysmic Engine
+# Sysmic Robotics Engine Robocup SSL
 
-## Building
+This project consists of two parts. The first part is the engine. The engine consists of the main program used to control the robots. On the other hand, the second part is an graphical interface intended to accelerate the development (debugging and testing) of the engine.
 
-First to create the build environment:
-```
-nix-shell
-```
+## 1 Requirements
 
-Then:
+For the development of this project the following requirements are needed 
 
-```
-./build.sh
-```
+### 1.1 Linux SO
 
-## Running the project
-```
-./run.sh
-```
+This project uses Linux for development, preferably Pop!_OS.
+
+Pop!_OS installation guide:
+
+https://www.youtube.com/watch?v=eol5O1qfWNs
+
+### 1.2 Grsim
+
+For this project we use grsim simulator, and it can be installed from the following link:
+
+https://github.com/RoboCup-SSL/grSim
 
-There is also a README in each project with the instructions to build them and run them separatly.
+### 1.3 Vscode
 
+For the code development we recommend encarecidamente utilizar Vscode code editor, In .vscode folder are the setting to run the engine program and gui program.
```

Comment:
For the code development we recommend encarecidamente utilizar Vscode code editor, In .vscode folder are the setting to run the engine program and gui program.

Está "encarecidamente utilizar" en español. Debería ser: we strongly recommend use Vscode code editor. Y luego, CREO que quedaría mejor poner:  the setting to run the engine and gui program.

Reply:
Hecho